dodiscover/replearning/gin.py

```diff
@@ -28,7 +29,7 @@ def __init__(self, indep_test_method="kci", alpha=0.05):
 
     def _causal_learn_to_pdag(self, cl_graph):
         """"""
-        def _extract_edgelists(adj_mat, names):
+        def _extract_edgelists(adj_mat: npt, names:str)-> tuple:
```

Comment:
`numpy.typing` is a module, not a type. Also `tuple` is not a type here either. See https://docs.python.org/3/library/typing.html and https://numpy.org/devdocs/reference/typing.html